Mediapipe.Net/Framework/Format/ImageFrame.cs

```diff
@@ -39,7 +39,7 @@ public ImageFrame(ImageFormat format, int width, int height, uint alignmentBound
         //     mediapipe::ImageFormat::Format format,
         //     int width, int height, int width_step, uint8* pixel_data,
         //     Deleter* deleter, mediapipe::ImageFrame** image_frame_out);
-        unsafe public ImageFrame(ImageFormat format, int width, int height, int widthStep, byte* pixelData)
+        unsafe public ImageFrame(ImageFormat format, int width, int height, int widthStep, ReadOnlySpan<byte> pixelData)
```

Comment:
Just changing this won't do, have you done a proper analysis on which classes needs to be redone?

Reply:
@RishiKumarRay no, i thought we have to just change this as it was mentioned in the issue by @Speykious

Reply:
To be absolutely clear, what we have to do is to *add a new constructor* that takes a `ReadOnlySpan<byte>` as parameter for the pixel data.
Might I ask (sorry if it comes out as rude), do you actually know any C#? Because I'm pretty sure that your IDE would tell you that this won't compile.

Reply:
@Speykious sorry i don't know c# , i thought we have to just refactor this

Reply:
A "good first issue" is an issue that is "good for newcomers" (as the description of the label says). I labelled it that way because I thought that someone who knows C# and is interested in contributing to this project could very easily create a new constructor that takes in a span instead of a pointer. But it certainly does **not** mean that it is good for someone who hasn't even touched the programming language at hand (except if it's about documentation or typos) or doesn't check that things are working properly.

We do need to *refactor* this, but while it is a relatively simple change, it is not just changing a type for another and your IDE should have told you that.